### PR TITLE
feat(uploader*.api.js): addInitialFiles() accepts non-successful files.

### PR DESCRIPTION
**Pull request by nalbion**
## Brief description of the changes 

If status is not provided, defaults to UPLOAD_SUCCESSFUL
My current project requires that files are scanned for viruses when uploaded, and when the user refreshes the page, the front end should query the back-end and be able to show files which were clean or contain viruses.  To do this, I need to call `addInitialFiles()` with some having `status = qq.status.UPLOAD_FAILED`

## What browsers and operating systems have you tested these changes on?
Chrome (not really applicable)

## Are all automated tests passing? 
check CI results

## Is this pull request against develop or some other non-master branch?
develop

### CODE REVIEW

File: docs/features/session.jmd
Comment (rnicholus) [inline docs/features/session.jmd]: Can you indicate that the accepted statues here are `qq.status.UPLOAD_SUCCESSFUL` and `qq.status.UPLOAD_FAILED`?

**Review by rnicholus — CHANGES_REQUESTED** [review]
A minor doc update, and please do write at least one unit test to exercise this new behavior. Once that it all done, I'll run some manual tests before merge. I'm hoping to release 5.14.0 soon, so this may target 5.15.0 instead.

**Comment by nalbion** [comment]
hmmm... `getNetUploads()` is documented:

> Get the number of items that have been successfully uploaded and have not been deleted

...What if a file has been uploaded but rejected?  (like my unit test was uploaded but rejected by Travis CI :wink: )

...answering my own question: seems that it would only be used for restricting the number of successful uploads

**Review by rnicholus — APPROVED** [review]

**Comment by rnicholus** [comment]
Looks good! I still need to do some manual testing. If that is successful, I'll merge into develop. I've marked this and your other recent PR for 5.15.0. I plan to release 5.14.0 in the next few days.

**Comment by nalbion** [comment]
thanks Ray!

**Comment by rnicholus** [comment]
Say, you've contributed quite a few PRs lately. I've been looking for a second maintainer for a while. I've been the only consistent maintainer of this library since 2012, and I could use a second dedicated person. Is this something you're interested in? If so, follow me on Twitter and we can discuss in more detail over DM.

**Comment by nalbion** [comment]
Hi Ray,
Thanks for the offer - I think FineUploader is the first repo I've contributed to which has so many forks that GitHub can't show the graph.  Being a maintainer of a project with such a large following would certainly look good on my CV, but I think at the moment I need to focus on what I've already got on my plate.
Nick 

    On Thursday, 16 February 2017, 16:03, Ray Nicholus <notifications@github.com> wrote:
 

 Say, you've contributed quite a few PRs lately. I've been looking for a second maintainer for a while. I've been the only consistent maintainer of this library since 2012, and I could use a second dedicated person. Is this something you're interested in? If so, follow me on Twitter and we can discuss in more detail over DM.—

**Comment by rnicholus** [comment]
Totally understand. If you change your mind, do let me know. When I was tapped to take over Fine Uploader back in 2012, I felt the same way, but it turned out to be a very worthwhile endeavor that opened up a great number of new doors. I wrote about this a bit at https://medium.com/@RayNicholus/disrupting-open-source-the-story-of-fine-uploader-80160eb557d9#.qiez71c3z.

**Comment by rnicholus** [comment]
I haven't forgotten about this. I'm in the process of switching jobs, so I have very little time to dedicate to Fine Uploader or other projects during this transition. I hope to get back to a normal OSS contribution schedule in the next couple months.

**Comment by nalbion** [comment]
I hope it's a good move for you

**Comment by nalbion** [comment]
@rnicholus please review merge of #1802

**Pull request closed without merging**
